### PR TITLE
Add Indian locale

### PR DESCRIPTION
**Pull request by pdvyas**

### CODE REVIEW

**Comment by hsoft** [comment]
Thanks for this PR, but not knowing Indian, I'm not too sure at what I look at. It's a locale based on english, but with an indian flavor? ... oh, nevermind, I can [google it](https://stuff.mit.edu/afs/athena/system/amd64_deb50/os/usr/share/i18n/locales/en_IN)

Is this language complete? Is the only difference between `en` and `en_IN` the `10**5` and `10**7` suffixes?

Otherwise, I would get rid of the tests (no need to re-test `en`) and I would add the locale code in the README.

**Comment by pdvyas** [comment]
Hey, sorry for not sending enough context.

In the Indian number system, we do not have "*illion" suffix. 10**5 is called 'lakh' and 10**7 is crore and that is all is needed. Eg, 10**13 is "10 lakh crore"

ref: http://en.wikipedia.org/wiki/Indian_Numbering_System#Names_of_numbers

Have removed the tests and added locale to readme. Let me know if I can improve it further.

File: num2words/lang_EN_IN.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+from __future__ import unicode_literals
+from .lang_EN import Num2Word_EN
+    
+class Num2Word_EN_IN(Num2Word_EN):
+    def set_high_numwords(self, high):
+        self.cards[10**7] = "crore"
+        self.cards[10**5] = "lakh"
+
+    def setup(self):
```
Comment (hsoft) [inline num2words/lang_EN_IN.py]: What is the purpose of those 3 lines? If I look at the code, the purpose of setting `self.high_numwords` seems to be so it is passed as a `high` argument in `set_high_numwords()`, but you don't use it in your override and the superclass' method is never called.

**Comment by hsoft** [comment]
I have tested it and it seems to work (although I'm not sure exactly what it's supposed to result in). I still have a question that I've asked inline, in the code.

**Comment by pdvyas** [comment]
Yes, you're right. It makes no difference and have removed it. Have also added a main() like in other files and expected output is,
```
For 15000, card is fifteen thousand;
	ord is fifteen thousandth; and
	ordnum is 15000th.
For 1500000, card is fifteen lakh;
	ord is fifteen lakhth; and
	ordnum is 1500000th.
For 15000000, card is one crore, fifty lakh;
	ord is one crore, fifty lakhth; and
	ordnum is 15000000th.
For 150000000, card is fifteen crore;
	ord is fifteen croreth; and
	ordnum is 150000000th.
For 1500000000, card is one hundred and fifty crore;
	ord is one hundred and fifty croreth; and
	ordnum is 1500000000th.
For 15000000000, card is one thousand, five hundred crore;
	ord is one thousand, five hundred croreth; and
	ordnum is 15000000000th.
For 150000000000, card is fifteen thousand crore;
	ord is fifteen thousand croreth; and
	ordnum is 150000000000th.
```

**Comment by hsoft** [comment]
Good, thanks! I'll make a PyPI release soon.

**Comment by pdvyas** [comment]
Awesome! Thanks

**Pull request merged**